jersey-tck/pom.xml

```diff
@@ -67,10 +145,140 @@
                         </goals>
                         <configuration>
                             <dependenciesToScan>jakarta.ws.rs:jakarta.ws.rs-tck</dependenciesToScan>
+                            <includes>
+                                 <include>**/tck/**/*IT</include>                            
+                            </includes>
+                            <systemPropertyVariables>
+                            <GLASSFISH_HOME>${project.build.directory}/glassfish6</GLASSFISH_HOME>
+                            <servlet_adaptor>org.glassfish.jersey.servlet.ServletContainer</servlet_adaptor>
+                            <webServerHost>localhost</webServerHost>
+                            <webServerPort>8080</webServerPort>
+                            </systemPropertyVariables>
+                             <environmentVariables>
+                                <GLASSFISH_HOME>${project.build.directory}/glassfish6</GLASSFISH_HOME>
+                             </environmentVariables> 
                         </configuration>
                     </execution>
                 </executions>
             </plugin>
+
+
+ <plugin>
+     <groupId>org.apache.maven.plugins</groupId>
+     <artifactId>maven-dependency-plugin</artifactId>
+     <version>3.2.0</version>
+     <executions>
+         <execution>
+             <id>unpack</id>
+             <phase>pre-integration-test</phase>
+             <goals>
+                 <goal>unpack</goal>
+             </goals>
+             <configuration>
+                 <artifactItems>
+                     <artifactItem>
+                         <groupId>org.glassfish.main.distributions</groupId>
```

Comment:
Is this essentially needed or can Jersey be tested without Glassfish?

Reply:
Using Glassfish seemed the easiest and working method to test jersey.  From what I understand if jersey implements jaxrs 3.1, the bootstrap api can be used to start the application in SE. I presume if there was an arquillian plugin to use jersey as container we could have used it instead of glassfish.

https://github.com/hantsy/jakartaee9-starter-boilerplate/tree/master/docs mentions ways to use other app servers as containers in arquillian (not specific to jersey).